pkg/urbit/noun/nock.c

```diff
@@ -495,14 +500,14 @@ _n_nock_on(u3_noun bus, u3_noun fol)
   X(KUTT, "kutt", &&do_kutt),  /* 84 */                                        \
   X(MUSM, "musm", &&do_musm),  /* 85 */                                        \
   X(KUSM, "kusm", &&do_kusm),  /* 86 */                                        \
-  X(MUTB, "mutb", &&do_mutb),  /* 87 */                                        \
-  X(MUTS, "muts", &&do_muts),  /* 88 */                                        \
-  X(MITB, "mitb", &&do_mitb),  /* 89 */                                        \
-  X(MITS, "mits", &&do_mits),  /* 90 */                                        \
-  X(KUTB, "kutb", &&do_kutb),  /* 91 */                                        \
-  X(KUTS, "kuts", &&do_kuts),  /* 92 */                                        \
-  X(KITB, "kitb", &&do_kitb),  /* 93 */                                        \
-  X(KITS, "kits", &&do_kits),  /* 94 */                                        \
+  X(MUTB, "mutb", &&do_mutb),  /* 87: c3_b */                                  \
+  X(MUTS, "muts", &&do_muts),  /* 88: c3_s */                                  \
+  X(MITB, "mitb", &&do_mitb),  /* 89: c3_b */                                  \
+  X(MITS, "mits", &&do_mits),  /* 90: c3_s, seems unused in codebase */        \
+  X(KUTB, "kutb", &&do_kutb),  /* 91: c3_b */                                  \
+  X(KUTS, "kuts", &&do_kuts),  /* 92: c3_s */                                  \
+  X(KITB, "kitb", &&do_kitb),  /* 93: c3_b */                                  \
+  X(KITS, "kits", &&do_kits),  /* 94: c3_s, seems unused in codebase */        \
```

Comment:
Some of these appear to be unused, but they are actually used through the index-size overflow mechanism.